### ISSUE TITLE
Drag and drop

### ISSUE THREAD

**Issue by dudijd** [linkwarden/linkwarden#307]
It would be nice if we could drag and drop links from the dashboard directly into collections. Would save a lot of clicks.

**Comment by snickers2k** [comment]
Nr1 and most important feature request of all , i guess
I wonder why this was not added in the first place.

Is there any roadmap for this feature? i'm stoked about linkwarden, but wonder if i should use it without this feature. is it planned already?

thank you

**Comment by jcolinger3** [comment]
+1 for Drag and Drop

**Comment by jcolinger3** [comment]
and batch editing

**Comment by Today20092** [comment]
> +1 for Drag and Drop

**Comment by SoCuul** [comment]
+1 this would be super nice for fluidity in moving stuff around

**Comment by couillonnade** [comment]
Drag and drop one or multiple items from one collection to another would be nice.

**Comment by marcmontecalvo** [comment]
I agree drag and drop would be great.

**Comment by dbkblk** [comment]
I also agree. It should be n°1 feature!

**Comment by AnAncientForce** [comment]
I was trying to find a replacement for Raindrop.io, but without this feature it feels really time consuming to organize and manage bookmarks, especially from a large collection.

**Comment by khanguyen74** [comment]
I'm interested in working on this, but I'm wondering, how would it improve the experience compared to simply selecting items and changing their collection in bulk? Since we already support that, what's the added value of drag-and-drop?

**Comment by silktrader** [comment]
> I'm interested in working on this, but I'm wondering, how would it improve the experience compared to simply selecting items and changing their collection in bulk? Since we already support that, what's the added value of drag-and-drop?

It's a significantly quicker operation to drag links to their target collection, folder.

When I add a link, through the browser extension (which doesn't provide a notification), I then wish to sort it.
Right now, I must click on the link's menu, then click on "Edit Link", then click on the "Collection" field, then click on the actual selection. That's a lot of clicks. Perhaps you can suggest a quicker process?

**Comment by khanguyen74** [comment]
> It's a significantly quicker operation to drag links to their target collection, folder.
> 
> When I add a link, through the browser extension (which doesn't provide a notification), I then wish to sort it. Right now, I must click on the link's menu, then click on "Edit Link", then click on the "Collection" field, then click on the actual selection. That's a lot of clicks. Perhaps you can suggest a quicker process?

Yeah, I actually agree now. I just made a quick drag-n-drop prototype and it felt really convenient. I’ve only been using Linkwarden for a few weeks so I don’t have a ton of links to manage yet, but I can see how drag-and-drop could enhance the overall experience.

**Comment by khanguyen74** [comment]
Good news, this feature has been merged, and hopefully will be released soon #1320

**Comment by daniel31x13** [comment]
Drag and drop is now available in [Linkwarden 2.12](https://github.com/linkwarden/linkwarden/releases/tag/v2.12.0), thanks to @khanguyen74!

**Comment by SoCuul** [comment]
> Drag and drop is now available in [Linkwarden 2.12](https://github.com/linkwarden/linkwarden/releases/tag/v2.12.0), thanks to [@khanguyen74](https://github.com/khanguyen74)!

Can this issue be closed in that case?
